Changelog.md

```diff
@@ -1,28 +1,32 @@
 #  Odo Changelog
 
-## Next
+## 2.3.0
```

Comment:
We're not sure when 2.3.0 will be released. Can we not put this here? `Next` sounds apt even though it was me who was confused. 😀

Reply:
That was the reason i put next there infact. Next for next version, and that will be updated by version bump pr